<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1313</width>
    <height>815</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>SlashGames</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resources.qrc">
    <normaloff>:/icons/universal/SlashGames.png</normaloff>:/icons/universal/SlashGames.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="1" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="supportLabel">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;TG:&lt;span style=&quot; color:#0000ff;&quot;/&gt;&lt;span style=&quot; color:#000000;&quot;&gt;SlashGames_support_bot&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>–ù–µ—Ç –Ω—É–∂–Ω–æ–π –∏–≥—Ä—ã? –ù–∞–ø–∏—à–∏—Ç–µ –Ω–∞–º!</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QScrollArea" name="gamesScrollArea">
      <property name="toolTip">
       <string/>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="gamesContainer">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1268</width>
         <height>660</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gamesGridLayout">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <property name="spacing">
         <number>20</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="rouletteButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>500</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#rouletteButton {
	background-color: #4CAF50;
    background-image: url(:/images/default/roulette.png);
    background-position: center;     
    border-radius: 25px;
	color: white; 
	font-size: 50px; 
	font-weight: bold; 
	padding-top: 240px
}

#rouletteButton:hover { 
    background-image: url(:/images/hover/roulette.png);   
}

#rouletteButton:pressed { 
    background-image: url(:/images/hover/roulette.png);  
 }</string>
          </property>
          <property name="text">
           <string>Roulette</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="slotsButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>500</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#slotsButton {
	background-color: #4CAF50;
    background-image: url(:/images/default/slots.png);
    background-position: center;     
    border-radius: 25px;
	color: white; 
	font-size: 50px; 
	font-weight: bold; 
	padding-top: 240px
}

#slotsButton:hover { 
    background-image: url(:/images/hover/slots.png);   
}

#slotsButton:pressed { 
    background-image: url(:/images/hover/slots.png);  
 }</string>
          </property>
          <property name="text">
           <string>Slots</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="addGameButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>500</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>In development...</string>
          </property>
          <property name="styleSheet">
           <string notr="true">#addGameButton { 
	background-color: #4CAF50; 
	font-size: 150px; 
	font-weight: bold; 
	color: #ffffff; 
	border: 2px solid #45a049; 
	border-radius: 25px;
}

#addGameButton:hover { 
	background-color: #45a049; 
}

#addGameButton:pressed { 
	background-color: #3d8b40; 
}</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="balance">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>24</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>–ë–∞–ª–∞–Ω—Å:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="balanceLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>24</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#balanceLabel {
	background: transparent;
	border: none;
}</string>
        </property>
        <property name="text">
         <string> 0üí≤</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="getBonusButton">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;–ü–æ–ª—É—á–∞—Ç—å –±–æ–Ω—É—Å –º–æ–∂–Ω–æ –∫–∞–∂–¥—ã–µ 3 —á–∞—Å–∞!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">#getBonusButton{
	background-color: #4CAF50; 
	color: #ffffff; 
	border: 2px solid #45a049; 
	border-radius: 25px; 
	padding: 15px; 
	font-size: 14px; 
	font-weight: bold; 
}

#getBonusButton:hover { 
	background-color: #45a049; 
}

#getBonusButton:pressed {
background-color: #3d8b40; 
}

#getBonusButton:disabled { 
background-color: #808080; 
border-color: #666666; 
}
</string>
        </property>
        <property name="text">
         <string>–ü–æ–ª—É—á–∏—Ç—å –ë–æ–Ω—É—Å!</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignLeft|Qt::AlignTop">
       <widget class="QLabel" name="balanceChangeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="3">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="5">
       <widget class="QToolButton" name="accountButton">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>–ê–∫–∫–∞—É–Ω—Ç</string>
        </property>
        <property name="styleSheet">
         <string notr="true">#accountButton {
    background: transparent;
    border: none;
}
#accountButton:hover {
    background: rgba(255, 255, 255, 0.1);
}
#accountButton:pressed {
    background: rgba(255, 255, 255, 0.2);
}
#accountButton::menu-indicator {
    image: none; 
    width: 0px;
   height: 0px;
}</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/light_mode/button_profile.png</normaloff>:/icons/light_mode/button_profile.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="FaQButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>FaQ and guide</string>
        </property>
        <property name="styleSheet">
         <string notr="true">#FaQButton{
    background: transparent;
    border: none;
}
#FaQButton:hover {
    background: rgba(255, 255, 255, 0.1);
}
#FaQButton:pressed {
    background: rgba(255, 255, 255, 0.2);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/light_mode/button_faq.png</normaloff>:/icons/light_mode/button_faq.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="7" alignment="Qt::AlignRight">
       <widget class="QPushButton" name="themeButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="toolTip">
         <string>–°–º–µ–Ω–∏—Ç—å –æ—Ñ–æ—Ä–º–ª–µ–Ω–∏–µ</string>
        </property>
        <property name="styleSheet">
         <string notr="true">#themeButton {
    background: transparent;
    border: none;
}

#themeButton:hover {
    background: rgba(255, 255, 255, 0.1);
}

#themeButton:pressed {
    background: rgba(255, 255, 255, 0.2);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/light_mode/button_toggle_theme.png</normaloff>:/icons/light_mode/button_toggle_theme.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="usernameLabel">
        <property name="styleSheet">
         <string notr="true">#usernameLabel {   
	color: #4CAF50; 
	font-size: 30px; 
	font-weight: bold; 
}</string>
        </property>
        <property name="text">
         <string>–ì–æ—Å—Ç—å</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1313</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
